pom.xml

```diff
@@ -1227,6 +1232,11 @@
         <artifactId>grpc-stub</artifactId>
         <version>${grpc.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>proto-google-common-protos</artifactId>
+        <version>2.9.6</version>
```

Comment:
This (and other) versions were required to avoid dependency convergence errors

Reply:
not an expert, but can we simply shade the one used in the 2 confluent input-format pom only?

Reply:
It could be an option, yes. We need to do an extensive study on how we are using shading. But I think that would be something we should do in another PR given shanding changes are difficult to verify and we already verified the proposal written here.